src/core/server/saved_objects/import/lib/validate_references.ts

```diff
@@ -76,7 +84,8 @@ export async function getNonExistingReferenceAsKeys(
 export async function validateReferences(
   savedObjects: Array<SavedObject<{ title?: string }>>,
   savedObjectsClient: SavedObjectsClientContract,
-  namespace?: string,
+  namespace: string | undefined,
+  importStateMap: ImportStateMap,
   retries?: SavedObjectsImportRetry[]
 ) {
```

Comment:
Same comment for this one.

Reply:
Done in 662b306cd2c9671b793d3cda12dbb808526e0b1e, I also removed the export here since it was only used for unit tests, and I refactored the tests cases / assertions accordingly.